src/EventStore.Core/ClusterVNode.cs

```diff
@@ -545,23 +551,143 @@ public class ClusterVNode :
 			perSubscrBus.Subscribe<SubscriptionMessage.PersistentSubscriptionTimerTick>(persistentSubscription);
 
 			// STORAGE SCAVENGER
-			var scavengerLogManager = new TFChunkScavengerLogManager(_nodeInfo.ExternalHttp.ToString(),
-				TimeSpan.FromDays(vNodeSettings.ScavengeHistoryMaxAge), ioDispatcher);
-			var storageScavenger = new StorageScavenger(db,
-				tableIndex,
-				readIndex,
-				scavengerLogManager,
-				vNodeSettings.AlwaysKeepScavenged,
-				!vNodeSettings.DisableScavengeMerging,
-				unsafeIgnoreHardDeletes: vNodeSettings.UnsafeIgnoreHardDeletes);
+			ScavengerFactory scavengerFactory;
+
+			var newScavenge = true;
```

Comment:
must be made configurable with config option

Reply:
actually in the end we dont want this to be configurable, in case subtle things happen if you interleave them. if we discover a problem with the new scavenge we'll fix it

Reply:
ok :+1: